commit 39162868afe6d51c1fb72d1b99defb51c7713fb6
Author: Diego Dompe <ddompe@users.noreply.github.com>
Date:   Thu Apr 12 20:02:40 2018 -0600

    Correct runtime issue for ARM64
    
    The getopt return is an int, and with the current code on ARM64 is being interpreted as a 255 given the opt is a char, causing to go into the default switch statement (and aborting the program silently).

diff --git src/ubridge.c src/ubridge.c
index fd2af71..daf46b0 100644
--- src/ubridge.c
+++ src/ubridge.c
@@ -334,7 +334,7 @@ int main(int argc, char **argv)
 {
   int hypervisor_tcp_port = 0;
   char *hypervisor_ip_address = NULL;
-  char opt;
+  int opt;
   char *index;
   size_t len;
 
